eventmesh-protocol-plugin/eventmesh-protocol-grpc/build.gradle

```diff
@@ -24,8 +24,8 @@ repositories {
     mavenCentral()
 }
 
-def grpcVersion = '1.43.2' // CURRENT_GRPC_VERSION
-def protobufVersion = '3.21.5'
+def grpcVersion = '1.63.0' // CURRENT_GRPC_VERSION
+def protobufVersion = '3.25.1'
 def protocVersion = protobufVersion
```

Comment:
I think we just need to define `grpcVersion` and `protobufVersion` once in the `build.gradle` file in the project root directory.

Reply:
I also think it's more appropriate to define it under the root directory. I will resubmit to fix the above issues.